.agent/workflows/rebase.md

```
@@ -0,0 +1,151 @@
+
+Because rebase rewrites commit SHAs:
+
+```bash
+git push --force-with-lease
+```
+
+## Output (required)
+
+Record exactly:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)